z390test/src/test/groovy/org/z390/test/RunRTBSAM.groovy

```diff
@@ -0,0 +1,41 @@
+package org.z390.test
+
+import static org.junit.jupiter.api.DynamicTest.dynamicTest
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.TestFactory
+
+class RunRTBSAM extends z390Test {
+
+    var options = ["SYSMAC(${basePath("mac")})"]
+
+    void run_test(String postfix) {
+        env = ['SNAPOUT': 'DUMMY', 'SYSUT2': basePath('bsam', "TEST${postfix}.TFV")]
+        int rc
+        rc = asmlg(basePath('bsam', "BLD${postfix}"), *options)
+        printOutput()
+        assert rc == 0
+        env.put('SYSUT1', basePath('bsam', "TEST${postfix}.TFV"))
+        rc = asmlg(basePath('bsam', "CHK${postfix}"), *options)
+        printOutput()
+        assert rc == 0
+        env = ['SNAPOUT': 'DUMMY',
+               'SYSUT1': basePath('bsam', "TEST${postfix}.TFV"),
+               'SYSUT2': basePath('bsam', "CPYR${postfix}.VES")
```

Comment:
Could this just be 
    env.put('SYSUT2', basePath('basm', "CPYR$Ppostifx}.VES")
or were the SNAPOUT and SYSUT1 values wiped out?

The bash/runrtbsam script did have to reset SYSUT1 for the final CPYR steps because of the order of the tests. However, you isolated the W and WL portions as separate tests, so the CPYR part has SYSUT1 set correctly after the BLD,CHK part.

Reply:
OK.. can see what you are saying.. just add the changed one in third step.. Yes.. you can do that.. but code is cheap and I almost prefer resetting the full env rather than relying on the previous steps values.. I'll change it to be consistent by using put but would also be happy redefining all envvars in each step for clarity.